### DISCUSSION TITLE
Autoplay can miss hits when shake mod is enabled

### DISCUSSION THREAD

**Discussion by NeonCantFindAUsername** [ppy/osu#17092] [Ideas]
Soo... in osu!standard: If you set the circle size to the minimum and make the notes shake very strong (I forgot how it's called) then the autobot will miss the notes. 
I don't really know if this is the right way to report a bug in osu!lazer... So yeah... Sorry

**Comment by peppy** [comment]
Auto isn't really guaranteed to hit perfectly on all mods. It's hard to ensure this. It can be achieved but it's a pretty low priority in the scheme of things.
